docs/source/user_guide/performance.rst

```diff
@@ -44,6 +44,26 @@ synchronizing over the end event (see :ref:`cuda_stream_event` for detail). Belo
 
 Additionally, :func:`cupyx.profiler.benchmark` runs a few warm-up runs to reduce timing fluctuation and exclude the overhead in first invocations.
 
+One-Time Overheads
```

Comment:
Just a nitpick: it seems in this file sections are separated by two empty lines.
```suggestion

One-Time Overheads
```